client/my-sites/sharing/connections/service-connected-accounts.jsx

```diff
@@ -1,87 +1,52 @@
 /**
  * External dependencies
  */
-import React, { Component, PropTypes } from 'react';
+import React, { PropTypes } from 'react';
 import { connect } from 'react-redux';
 import { identity } from 'lodash';
 import { localize } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
-import Connection from './connection';
 import { recordGoogleEvent } from 'state/analytics/actions';
 
-class SharingServiceConnectedAccounts extends Component {
-	static propTypes = {
-		connections: PropTypes.array,               // Set of connections for the service
-		isDisconnecting: PropTypes.bool,            // Whether a disconnect request is pending
-		isRefreshing: PropTypes.bool,               // Whether a connection refresh is pending
-		onAddConnection: PropTypes.func,            // Handler to invoke when adding a new connection
-		onRefreshConnection: PropTypes.func,        // Handler to invoke when refreshing a connection
-		onRemoveConnection: PropTypes.func,         // Handler to invoke when removing an existing connection
-		onToggleSitewideConnection: PropTypes.func, // Handler to invoke when toggling a connection to be shared sitewide
-		recordGoogleEvent: PropTypes.func,          // Redux action to track an event
-		service: PropTypes.object.isRequired,       // The service object
-		translate: PropTypes.func,
+const SharingServiceConnectedAccounts = ( { children, onAddConnection, recordEvent, service, translate } ) => {
+	const connectAnother = () => {
+		onAddConnection( service );
+		recordEvent( 'Sharing', 'Clicked Connect Another Account Button', service.ID );
```

Comment:
I ended up creating a new method within `SharingService` because `connect()` is not only called from the "Connect Another" button, if there's a better way please let me know